### ISSUE TITLE
libreoffice 4.6.2

### ISSUE THREAD

**Issue by Extarys** [feren-OS/Feren-OS-Dependency-Issue-Reporting#14]
**Make sure you have the default repository set in Feren OS**
If you have any other repositories enabled in Feren OS outside of the default set of repositories, then this might be the cause of your package's dependency issue.

**What is the name of the troublesome package?**
Please put the name of the package here: Libreoffice-common 6.4.2

**What is the output of an apt-get install attempt for this package?**
Please copy and paste the output of apt-get install (troublesome package name here) below in the space between the two triple-`s:
```
$ sudo apt upgrade libreoffice-common
Reading package lists... Done
Building dependency tree       
Reading state information... Done
You might want to run 'apt --fix-broken install' to correct these.
The following packages have unmet dependencies:
 libreoffice-base : Depends: libreoffice-common (>= 1:6.4.0~beta1-2~) but 1:6.3.5~rc2-0ubuntu0.18.04.1~lo1 is installed
                    Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                    Depends: libuno-cppuhelpergcc3-3 (>= 4.0.0~alpha) but it is not installed
                    Depends: libuno-sal3 (>= 5.1.0~alpha) but it is not installed
                    Depends: libuno-salhelpergcc3-3 (>= 1.4.0) but it is not installed
                    Depends: uno-libs-private but it is not installed
                    Breaks: libreoffice-common (< 1:6.4.2~rc1~) but 1:6.3.5~rc2-0ubuntu0.18.04.1~lo1 is installed
 libreoffice-base-core : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                         Depends: libuno-cppuhelpergcc3-3 (>= 5.3.0~alpha) but it is not installed
                         Depends: libuno-sal3 (>= 5.1.0~alpha) but it is not installed
                         Depends: libuno-salhelpergcc3-3 (>= 1.4.0) but it is not installed
                         Depends: uno-libs-private but it is not installed
 libreoffice-base-drivers : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                            Depends: libuno-cppuhelpergcc3-3 (>= 5.3.0~alpha) but it is not installed
                            Depends: libuno-sal3 (>= 5.3.0~alpha) but it is not installed
                            Depends: libuno-salhelpergcc3-3 (>= 1.4.0) but it is not installed
                            Depends: uno-libs-private but it is not installed
 libreoffice-calc : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                    Depends: libuno-cppuhelpergcc3-3 (>= 4.0.0~alpha) but it is not installed
                    Depends: libuno-sal3 (>= 5.3.0~alpha) but it is not installed
                    Depends: libuno-salhelpergcc3-3 (>= 3.6.0~beta) but it is not installed
                    Depends: uno-libs-private but it is not installed
                    Breaks: libreoffice-common (< 1:6.4.2~rc1~) but 1:6.3.5~rc2-0ubuntu0.18.04.1~lo1 is installed
 libreoffice-common : Breaks: libreoffice-core (>= 1:6.4~) but 1:6.4.2-0ubuntu0.18.04.2 is installed
                      Breaks: libreoffice-style-tango (>= 1:6.4~) but 1:6.4.2-0ubuntu0.18.04.2 is installed
 libreoffice-core : Depends: libreoffice-common (> 1:6.4.2) but 1:6.3.5~rc2-0ubuntu0.18.04.1~lo1 is installed
                    Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                    Depends: libuno-cppuhelpergcc3-3 (>= 5.3.0~alpha) but it is not installed
                    Depends: libuno-sal3 (>= 6.4.0~beta) but it is not installed
                    Depends: libuno-salhelpergcc3-3 (>= 3.6.0~beta) but it is not installed
                    Depends: uno-libs-private but it is not installed
 libreoffice-draw : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                    Depends: libuno-cppuhelpergcc3-3 (>= 5.3.0~alpha) but it is not installed
                    Depends: libuno-sal3 (>= 5.3.0~alpha) but it is not installed
                    Depends: libuno-salhelpergcc3-3 (>= 3.6.0~beta) but it is not installed
                    Depends: uno-libs-private but it is not installed
                    Breaks: libreoffice-common (< 1:6.4.2~rc1~) but 1:6.3.5~rc2-0ubuntu0.18.04.1~lo1 is installed
 libreoffice-gnome : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                     Depends: libuno-cppuhelpergcc3-3 (>= 4.0.0~alpha) but it is not installed
                     Depends: libuno-sal3 (>= 5.1.0~alpha) but it is not installed
                     Depends: libuno-salhelpergcc3-3 (>= 1.4.0) but it is not installed
                     Depends: uno-libs-private but it is not installed
 libreoffice-gtk3 : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                    Depends: libuno-cppuhelpergcc3-3 (>= 4.0.0~alpha) but it is not installed
                    Depends: libuno-sal3 (>= 5.1.0~alpha) but it is not installed
                    Depends: uno-libs-private but it is not installed
 libreoffice-impress : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                       Depends: libuno-cppuhelpergcc3-3 (>= 5.3.0~alpha) but it is not installed
                       Depends: libuno-sal3 (>= 5.3.0~alpha) but it is not installed
                       Depends: libuno-salhelpergcc3-3 (>= 1.4.0) but it is not installed
                       Depends: uno-libs-private but it is not installed
                       Breaks: libreoffice-common (< 1:6.4.2~rc1~) but 1:6.3.5~rc2-0ubuntu0.18.04.1~lo1 is installed
 libreoffice-java-common : Depends: libjuh-java but it is not installed
                           Depends: libjurt-java but it is not installed
                           Depends: libridl-java but it is not installed
                           Depends: libunoil-java but it is not installed
                           Depends: libunoloader-java but it is not installed
 libreoffice-math : Depends: libreoffice-core (= 1:6.3.5~rc2-0ubuntu0.18.04.1~lo1) but 1:6.4.2-0ubuntu0.18.04.2 is installed
 libreoffice-report-builder : Breaks: libreoffice-common (< 1:6.4.2~rc1~) but 1:6.3.5~rc2-0ubuntu0.18.04.1~lo1 is installed
 libreoffice-report-builder-bin : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                                  Depends: libuno-cppuhelpergcc3-3 (>= 4.0.0~alpha) but it is not installed
                                  Depends: libuno-sal3 (>= 5.1.0~alpha) but it is not installed
                                  Depends: libuno-salhelpergcc3-3 (>= 1.4.0) but it is not installed
                                  Depends: uno-libs-private but it is not installed
 libreoffice-sdbc-firebird : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                             Depends: libuno-cppuhelpergcc3-3 (>= 5.3.0~alpha) but it is not installed
                             Depends: libuno-sal3 (>= 5.1.0~alpha) but it is not installed
                             Depends: libuno-salhelpergcc3-3 (>= 1.4.0) but it is not installed
                             Depends: uno-libs-private but it is not installed
 libreoffice-sdbc-mysql : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                          Depends: libuno-cppuhelpergcc3-3 (>= 5.3.0~alpha) but it is not installed
                          Depends: libuno-sal3 (>= 4.2.0~rc1) but it is not installed
                          Depends: libuno-salhelpergcc3-3 (>= 1.4.0) but it is not installed
                          Depends: uno-libs-private but it is not installed
 libreoffice-sdbc-postgresql : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                               Depends: libuno-cppuhelpergcc3-3 (>= 4.0.0~alpha) but it is not installed
                               Depends: libuno-sal3 (>= 4.1.0~alpha) but it is not installed
                               Depends: libuno-salhelpergcc3-3 (>= 1.4.0) but it is not installed
                               Depends: uno-libs-private but it is not installed
 libreoffice-writer : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
                      Depends: libuno-cppuhelpergcc3-3 (>= 5.3.0~alpha) but it is not installed
                      Depends: libuno-sal3 (>= 5.3.0~alpha) but it is not installed
                      Depends: libuno-salhelpergcc3-3 (>= 1.4.0) but it is not installed
                      Depends: uno-libs-private but it is not installed
                      Breaks: libreoffice-common (< 1:6.4.2~rc1~) but 1:6.3.5~rc2-0ubuntu0.18.04.1~lo1 is installed
 python3-uno : Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
               Depends: libuno-cppuhelpergcc3-3 (>= 5.3.0~alpha) but it is not installed
               Depends: libuno-sal3 (>= 5.4.0~alpha) but it is not installed
               Depends: libuno-salhelpergcc3-3 (>= 3.6.0~beta) but it is not installed
               Depends: uno-libs-private but it is not installed
 ure : Depends: uno-libs-private (= 1:6.4.2-0ubuntu0.18.04.2) but it is not installed
       Depends: libuno-cppu3 (>= 4.4.0~alpha) but it is not installed
       Depends: libuno-cppuhelpergcc3-3 (>= 4.0.0~alpha) but it is not installed
       Depends: libuno-purpenvhelpergcc3-3 (>= 1.4.0) but it is not installed
       Depends: libuno-sal3 (>= 5.3.0~alpha) but it is not installed
       Depends: libuno-salhelpergcc3-3 (>= 3.6.0~beta) but it is not installed
       Recommends: libjuh-java but it is not installed
       Recommends: libjurt-java but it is not installed
       Recommends: libridl-java but it is not installed
       Recommends: libunoloader-java but it is not installed
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution)
```
The error started after I had the notification in the update center about libreoffice. This is a brand new install (I was on Mint before). Yesterday I did all the upgrades and today I had this one appeared.
```
$ sudo apt --fix-broken install
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Correcting dependencies... Done
The following package was automatically installed and is no longer required:
  virt-what
Use 'sudo apt autoremove' to remove it.
The following additional packages will be installed:
  libjuh-java libjurt-java libreoffice-common libreoffice-math libridl-java libuno-cppu3 libuno-cppuhelpergcc3-3 libuno-purpenvhelpergcc3-3 libuno-sal3
  libuno-salhelpergcc3-3 libunoil-java libunoloader-java uno-libs-private
The following packages will be REMOVED:
  uno-libs3
The following NEW packages will be installed:
  libjuh-java libjurt-java libridl-java libuno-cppu3 libuno-cppuhelpergcc3-3 libuno-purpenvhelpergcc3-3 libuno-sal3 libuno-salhelpergcc3-3 libunoil-java libunoloader-java
  uno-libs-private
The following packages will be upgraded:
  libreoffice-common libreoffice-math
2 upgraded, 11 newly installed, 1 to remove and 0 not upgraded.
21 not fully installed or removed.
Need to get 0 B/29.2 MB of archives.
After this operation, 32.8 MB disk space will be freed.
Do you want to continue? [Y/n] 
(Reading database ... 271550 files and directories currently installed.)
Preparing to unpack .../libreoffice-math_1%3a6.4.2-0ubuntu0.18.04.2_amd64.deb ...
Unpacking libreoffice-math (1:6.4.2-0ubuntu0.18.04.2) over (1:6.3.5~rc2-0ubuntu0.18.04.1~lo1) ...
dpkg: error processing archive /var/cache/apt/archives/libreoffice-math_1%3a6.4.2-0ubuntu0.18.04.2_amd64.deb (--unpack):
 trying to overwrite '/usr/lib/libreoffice/share/config/soffice.cfg/modules/smath/menubar/menubar.xml', which is also in package libreoffice-common 1:6.3.5~rc2-0ubuntu0.18.04.1~lo1
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
Restoring backup of /usr/share/doc/libreoffice-math ...
Errors were encountered while processing:
 /var/cache/apt/archives/libreoffice-math_1%3a6.4.2-0ubuntu0.18.04.2_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Thank you for helping me fix as many dependency issues as possible. When you're ready to report, click 'Submit new issue'. You can be contacted by The Developer or others via replies to this issue in the future, so please be sure to check GitHub Notifications during the time this issue is open, otherwise this issue could be closed due to inactivity from those suffering from this dependency issue.

**Comment by dominichayesferen** [comment]
Both of these troublesome packages are from LibreOffice's Official PPA for Ubuntu (and therefore Feren OS) so you'll have to wait on them for now to fix their official repositories, though it'd probably be for the best if you also report this issue to them so they know about it sooner.

TL;DR: Either try again later when they've fixed it upstream or do `sudo dpkg --purge libreoffice-math libreoffice` to get rid of LibreOffice Math to fix this package conflict immediately.
